RadarSDK/RadarReplayBuffer.m

```diff
@@ -42,6 +43,10 @@ - (void)writeNewReplayToBuffer:(NSMutableDictionary *)replayParams {
     // add new replay to buffer
     RadarReplay *radarReplay = [[RadarReplay alloc] initWithParams:replayParams];
     [mutableReplayBuffer addObject:radarReplay];
+
+    // persist buffer
+    NSData *replaysData = [NSKeyedArchiver archivedDataWithRootObject:mutableReplayBuffer];
+    [[NSUserDefaults standardUserDefaults] setObject:replaysData forKey:@"radar-replays"];
```

Comment:
just want to be cognizant that we're rewriting the entire buffer to persistent storage each time here

Reply:
We could have a manifest of entries but that's probably a premature optimization at this point.